packages/api-server/src/api/admin.ts

```diff
@@ -61,18 +62,19 @@ router.post('/addSubdomain', async (req, res) => {
         throw new Error('DNS Config Error')
       }
     }
+    */
 
     await db.dbQuery(
-      `UPDATE servers SET subdomain = $1, is_public = $2, discord_url = $3 WHERE id = $4`,
-      [body.subdomain, !!body.isPublic, discordUrl, body.id]
+      `UPDATE servers SET path = $1, is_public = $2, discord_url = $3 WHERE id = $4`,
```

Comment:
`instance_path`

They body can be named anything, but the SQL needs to match the definition of the table.